src/views/App/index.tsx

```diff
@@ -80,6 +81,8 @@ const AppRoutes = () => {
       <Route path="/signin" exact component={Login} />
       <SecureRoute path="/user-diagnostics" component={UserInfo} />
 
+      <Route path="/unfollow" exact component={Unfollow} />
```

Comment:
This should be a `SecureRoute` component.  Right now our dev login doesn't support maintaining an original url after login, but Okta does.  It should not be a problem in prod/once we get IdP connected